tensorflow/compiler/tf2tensorrt/convert/utils.cc

```diff
@@ -157,6 +161,10 @@ Status TfTypeToTrtType(DataType tf_type, nvinfer1::DataType* trt_type) {
     case DT_INT32:
       *trt_type = nvinfer1::DataType::kINT32;
       break;
+    // TODO add #ifdef bool is available
```

Comment:
Do you need to handle this in this PR?
If you will handle this is a follow up PR, please fix the format of the TODO description:

// TODO(your username):  end with a period.